packages/svelte/src/lib/component/shared.ts

```diff
@@ -0,0 +1,108 @@
+import type {
+  ExpandedRouteConfig,
+  UploadThingError,
+} from "@uploadthing/shared";
+import type { UploadFileResponse } from "uploadthing/client";
+import type {
+  ErrorMessage,
+  FileRouter,
+  inferEndpointInput,
+  inferErrorShape,
+} from "uploadthing/server";
+
+export const generatePermittedFileTypes = (config?: ExpandedRouteConfig) => {
```

Comment:
Fyi, These helper functions will be exported from uplodthing/client once the solid theming pr is merged

Reply:
merged now